config/routes.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
-DiscourseCustomTopicLists::Engine.routes.draw { get "lists/:topic_list_name" => "lists#show" }
+DiscourseCustomTopicLists::Engine.routes.draw do
+  get "list/:topic_list_name",
+      to: redirect { |params, request| "lists/#{params[:topic_list_name]}" }
```

Comment:
Just curious, why is this redirect necessary?

Reply:
While I was testing, I thought it could be an excellent addition, people might forget the `s` in `/lists/<name>`.

But after sleeping on it does not make that much sense, I'll remove it!